src/main/java/de/retest/web/selenium/AutocheckingRecheckDriver.java

```diff
@@ -0,0 +1,93 @@
+package de.retest.web.selenium;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+import de.retest.web.RecheckWebImpl;
+
+public class AutocheckingRecheckDriver extends RecheckDriver {
```

Comment:
AFAIK the "official" mechanism for this would be to use `EventFiringWebDriver` ([example](https://seleniumeasy.com/selenium-tutorials/webdriver-event-listener-example/)). But since we already adopted the approach from `RecheckDriver`, I think it doesn't make sense to switch. (Honestly, I also don't know which is "better".)

Reply:
`EventFiringWebDriver` also has `EventFiringKeyboard` and `EventFiringMouse` (even `EventFiringTouch`). So I believe we also have to overwrite at least `getKeyboard()` and `getMouse()` and create wrappers for these?